License2Deploy/rolling_deploy.py

```diff
@@ -52,6 +52,7 @@ def __init__(self,
     self.health_wait = health_wait
     self.only_new_wait = only_new_wait
     self.existing_instance_ids = []
+    self.old_desired_capacity = 2
     self.new_desired_capacity = None
```

Comment:
this is a dead gobal variable as you've refactored to only be used within method scope